pynest/nest/tests/test_connect_arrays.py

```diff
@@ -20,29 +20,52 @@
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
 
 import unittest
-import nest
 import numpy as np
 
+import nest
+from nest.lib.hl_api_exceptions import NESTErrors
```

Comment:
`NESTErrors` is already available, and used, as `nest.kernel.NESTErrors`. It's ok to import this, but the usage should be consistent.